bundles/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/common/HomematicConfig.java

```diff
@@ -167,10 +176,10 @@ public boolean isValid() {
     }
 
     /**
-     * Returns the BIN-RPC url.
+     * Returns true, if HomematicIP support is enabled.
      */
-    public String getBinRpcCallbackUrl() {
-        return "binary://" + callbackHost + ":" + callbackPort;
+    public boolean isHomematicIpEnabled() {
+        return homematicIpEnabled;
     }
```

Comment:
This is removing a pre-existing public method. Should probably leave this in and mark it as deprecated.